controllers/productController.js

```diff
@@ -0,0 +1,17 @@
+const { productService } = require("../services");
+const { asyncWrap } = require("../middleware/errorControl");
+
+const detailProduct = asyncWrap(async (req, res) => {
+    const { productId } = req.params;
+    if ( !productId ) {
+        const err = new Error("KEY_ERROR");
+        err.statusCode = 400;
+        throw err;
+    }
+    const [ detail, review ] = await productService.detailProduct( productId );
+    return res.status(200).json({ product: detail, review: review})
```

Comment:
가독성을 위해서, 간격을 추가해주세요.

```suggestion
    const { productId } = req.params;
    
    if ( !productId ) {
        const err = new Error("KEY_ERROR");
        err.statusCode = 400;
        throw err;
    }
    
    const [ product, review ] = await productService.getProduct(productId);
    
    return res.status(200).json({ product: product, review: review})
```